commit 82ce5e7b1c971f3645f767a947dacc01bbd92ead
Author: dyep <dyep49@gmail.com>
Date:   2019-03-28T10:34:42-07:00 (6 months ago)
    
    TRITON-1358 Logarchiver logset manta path is incorrect
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/etc/logsets.json b/etc/logsets.json
index ea35280..f9fe1fc 100644
--- a/etc/logsets.json
+++ b/etc/logsets.json
@@ -3,7 +3,7 @@
     "name": "firewall_logs",
     "search_dirs": [ "/var/log/firewall" ],
     "regex": "^/var/log/firewall/([0-9a-f]{8}\\-[0-9a-f]{4}\\-[0-9a-f]{4}\\-[0-9a-f]{4}\\-[0-9a-f]{12})/([0-9a-f]{8}\\-[0-9a-f]{4}\\-[0-9a-f]{4}\\-[0-9a-f]{4}\\-[0-9a-f]{12})/([0-9]+)-([0-9]+)-([0-9]+)T([0-9]+):([0-9]+):([0-9]+)\\.log.gz$",
-    "manta_path": "/%U/reports/firewall-logs/$2/#y/#m/#d/#y-%m-%dT%H:%M:%S.log.gz",
+    "manta_path": "/%U/reports/firewall-logs/#y/#m/#d/$2/#y-#m-#dT#H:#M:#S.log.gz",
     "customer_uuid": "$1",
     "date_string": {
       "y": "$3", "m": "$4", "d": "$5",
